### PR TITLE
update buildout to modernise tests

### PR DESCRIPTION
**Pull request by djay**

### CODE REVIEW

File: test-4.3.x.cfg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[buildout]
+extends =
+    https://raw.githubusercontent.com/collective/buildout.plonetest/master/test-5.0.x.cfg
```
Comment (tomgross) [inline test-4.3.x.cfg]: shouldn't this be 4.3?

File: .travis.yml [hunk 2]
```diff
@@ -1,21 +1,57 @@
...
-  - mkdir -p buildout-cache/downloads
-  - buildout -c travis.cfg -N -q -t 3
-script: bin/test
-services:
-  - xvfb
+  - bin/buildout -N -t 20 -c test-$PLONE_VERSION.cfg
+  - pip install zest.pocompile
+  - pocompile src
+before_script:
+  - 'export DISPLAY=:99.0'
```
Comment (tomgross) [inline .travis.yml]: I believe this is not necessary: https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-xvfb-to-run-tests-that-require-a-gui

File: test-4.3.x.cfg (hunk 1 above)
Comment (djay) [reply test-4.3.x.cfg]: fixed

File: .travis.yml (hunk 2 above)
Comment (djay) [reply .travis.yml]: removed

**Pull request merged**